### PR TITLE
refactor: navtree visual enhancements

### PR DESCRIPTION
**Pull request by wittjosiah**
- add padding to space create/join actions
- hide drag indicator when not dragging over spaces/objects
- add padding to top of navtree for focus ring

image

### CODE REVIEW

File: packages/ui/react-ui-list/src/components/Tree/types.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export const ItemSchema = S.mutable(
     icon: S.optional(S.String),
     disabled: S.optional(S.Boolean),
     className: S.optional(S.String),
```
Comment (richburdon) [inline packages/ui/react-ui-list/src/components/Tree/types.ts:24]: consider the following if you think we're going to want to further customize:

```
classNames:
  root
  heading
```

**Review by richburdon — APPROVED** [review]

File: packages/plugins/plugin-navtree/src/NavTreePlugin.tsx [hunk 2]
```diff
@@ -89,6 +89,8 @@ export const NavTreePlugin = (): PluginDefinition<NavTreePluginProvides> => {
       icon: node.properties.icon,
       disabled: node.properties.disabled,
       testId: node.properties.testId,
+      className: node.properties.className,
```
[the remark this answers is not in the document]
Comment (thure) [reply packages/plugins/plugin-navtree/src/NavTreePlugin.tsx:92]: I feel super weird about putting something so utterly presentational in graph nodes, but if you think this is the best solution I will defer.

**Review by thure — APPROVED** [review]
👌

File: packages/plugins/plugin-navtree/src/NavTreePlugin.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (wittjosiah) [reply packages/plugins/plugin-navtree/src/NavTreePlugin.tsx:92]: I would actually say that graph nodes are presentational. Their main properties are labels and icons and they map directly into the ui. Previously we also had palette as a property and this sort of replaces that for now.

If we figure out a better abstraction later for the graph node properties which maps to the tree item class names then I'd be happy with that as well.

**Pull request merged**